User_Service/Database/compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.8'
+
+services:
+  postgres:
+    image: postgres:14
+    container_name: web_app_container
+    environment:
+      POSTGRES_USER: furqan
+      POSTGRES_PASSWORD: furqan
+      POSTGRES_DB: User
+    ports:
+      - "5430:5432"
+    volumes:
+      - postgres_data:/backup
+    networks:
+      - backend
+
+volumes:
+  postgres_data:
+
+networks:
+  backend:
```

Comment:
can keep it with, no need for separate folder

Reply:
moved the compose file outside the folder